api/pkg/cmd/stack.go

```diff
@@ -22,80 +16,35 @@ type StackManager interface {
 }
 
 type Stack struct {
-	db  StackManager
-	ecs StackManager
-	eks StackManager
+	// TODO: see note below about eventually read and writing stack information to a database
+	// keeping this as a reminder
+	//db StackManager
 }
 
 func MakeStack(db *dbutil.DB) StackManager {
 	return &Stack{
 		// DB is not currently used since this is currently just a read interface for the old data locations
 		// but we should keep this here so it's easy to set up later when we want to move the data
-		db:  MakeStackBackendDB(db),
-		ecs: &StackBackendECS{},
-		eks: &StackBackendEKS{},
+		//db: MakeStackBackendDB(db),
 	}
 }
 
 func (s Stack) GetAppStacks(ctx context.Context, payload model.AppStackPayload) ([]*model.AppStackResponse, error) {
-	switch payload.TaskLaunchType {
-	case "k8s":
-		return s.eks.GetAppStacks(ctx, payload)
-	case "fargate":
-		return s.ecs.GetAppStacks(ctx, payload)
-	default:
-		logrus.Fatal("Must specify a Launch Type as either k8s or fargate")
-	}
-	return nil, nil
-}
-
-func parseParamToStacklist(paramOutput string) ([]string, error) {
-	var stacklist []string
-	err := json.Unmarshal([]byte(paramOutput), &stacklist)
-	return stacklist, errors.Wrap(err, "could not parse json")
-}
-
-func enrichStacklistMetadata(ctx context.Context, stacklist []string, payload model.AppStackPayload, integrationSecret *config.IntegrationSecret) ([]*model.AppStackResponse, error) {
-	workspaceRepo := workspace_repo.NewWorkspaceRepo(
-		integrationSecret.Tfe.Url,
-		integrationSecret.Tfe.Org,
-	).WithTFEToken(setup.GetConfiguration().TFE.Token)
-	wg := sync.WaitGroup{}
-
-	stackInfos := make([]*model.AppStackResponse, len(stacklist))
-	for i, stackName := range stacklist {
-		wg.Add(1)
-		go func(id int, stackName string) {
-			defer wg.Done()
-
-			stackInfo, err := getStackInfo(ctx, payload, stackName, workspaceRepo)
-			if err != nil {
-				stackInfo.Error = err.Error()
-			}
-			stackInfos[id] = stackInfo
-		}(i, stackName)
-	}
-	wg.Wait()
-
-	return stackInfos, nil
-}
-
-func getStackInfo(ctx context.Context, payload model.AppStackPayload, stackName string, workspaceRepo workspace_repo.WorkspaceRepoIface) (*model.AppStackResponse, error) {
-	stack := &model.AppStackResponse{
-		AppMetadata: *model.NewAppMetadata(payload.AppName, payload.Environment, stackName),
-	}
-	workspace, err := workspaceRepo.GetWorkspace(ctx, fmt.Sprintf("%s-%s", payload.AppMetadata.Environment, stackName))
+	happyClient, err := utils.MakeHappyClient(ctx, payload.AppMetadata)
```

Comment:
we not have a `MakeHappyClient` that is implemented in the CLI so that the API requests can have a happy client based on the query parameters passed from the user.